tests/mesos_test.py

```diff
@@ -249,6 +249,17 @@ def test_log_event_error(self):
             assert mock_log_event.called
         assert self.task.state == MesosTask.RUNNING
 
+    def test_get_event_logger_add_unique_handlers(self):
+        """
+        Ensures that only a single handler (for stderr) is added to the
+        MesosTask event logger, to prevent duplicate log output.
+        """
+        log1 = self.task.get_event_logger()
+        log2 = self.task.get_event_logger()
+
+        assert len(log1.handlers) == len(log2.handlers)
```

Comment:
Out of curiosity, does "assert log1 == log2" work here?

Reply:
Yes, it would. `self.task.get_event_logger()' always returns the same logger (since all MesosTasks share the same logger). So, actually, `assert len(log1.handlers) == len(log2.handlers)` will always be `True` since log1 and log2 are the same reference. I think I will remove this check and just keep the subsequent length = 1 check.